README.md

```diff
@@ -159,7 +159,7 @@ Type: `string` or `string[]`
 Default value: `[]`
 
 Whether to fail if any message key is not prefixed by the given prefix, or if multiple, one of the
-given prefices.
+given prefixes.
```

Comment:
This isn't a typo. https://en.wiktionary.org/wiki/prefices

Reply:
It says "(nonstandard)" on Wiktionary for a reason :P but sure